tensorboard/components/tf_ng_tensorboard/types/api.ts

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+export enum ActiveDashboardsLoadState {
+  NOT_LOADED,
+  LOADED,
+  FAILED,
```

Comment:
Maybe s/FAILED/FAILED_TO_LOAD/ to be clearer in meaning?

Reply:
(1) this is a copy/branch from https://github.com/tensorflow/tensorboard/blob/2b96c2a18da8cfe5f387cc07d03e8227706ca914/tensorboard/components/tf_tensorboard/registry.ts#L16-L20 which is not to say it is correct thing to keep but (2) because the word "Load" is included as part of the enum, it is more than clear already (I've seen repeated words in enum in Java too much to now hate it).